content/applications/finance/accounting/bank/reconciliation/use_cases.rst

```diff
@@ -1,105 +1,92 @@
+payment information. When done, click :guilabel:`Create Payment`. The payment status of the document
+will change to "In Payment".
 
-.. image:: use_cases/use03.png
+.. image:: use_cases/register-payment.png
    :align: center
+   :alt: The Register Payment button.
 
-Now we can always find the payment details in the Invoice by clicking on the
-:menuselection:`Info --> Open Payment`.
-
-.. image:: use_cases/use04.png
+.. image:: use_cases/pop-up-window.png
    :align: center
+   :alt: Pop-up window.
 
-The invoice has been paid and **the reconciliation has been done
-automatically.**
+Bank statement matching
+-----------------------
 
-Case 2: Bank statements reconciliations
----------------------------------------
+When the payment is registered, you must now match it with its corresponding bank statement. Head to
+the **Accounting Dashboard**, click on the **Bank** journal, and create a new bank statement. If you
+have synchronized your Odoo instance with your bank account, select the imported bank statement that
+matches your invoice or bill.
 
-We start at our issued Invoice of 3000 euros for Smith & Co. Let's also
-assume that other Invoices are open for different customers.
-
-.. image:: use_cases/use05.png
+.. image:: use_cases/bank-statements.png
    :align: center
+   :alt: Bank statements menu.
 
-We receive our bank statement and find that not only the invoice issued to Smith & Co has
-been paid, but the one to Buzz of 92 euros as well.
+Once you have populated the required fields, click on **Post** and then **Reconcile**. You will land
+on the reconciliation page, where you can match the payment with the corresponding bank statement.
+Odoo automatically suggests matches based on the **Label**, **Partner**, and **Amount** entered upon
+the creation or import of the bank statement. However, if no matches are suggested or if the match
+does not correspond with its counterpart entry, you can always change the partner, or select another
+payment from the list by clicking on the blue line (cf. screenshot below).
 
-**Import** or **Create** the bank statements. Please refer to the
-documents from the Bank Feeds section.
+Finally, click on :guilabel:`Validate` to reconcile both entries. The payment status ribbon will now
+switch to "Paid".
 
-.. image:: use_cases/use06.png
+.. image:: use_cases/reconciliation.png
    :align: center
+   :alt: The reconciliation page.
 
-On the dashboard, click on **Reconcile # Items**
+.. _bypass:
 
-.. image:: use_cases/use07.png
-   :align: center
+"In payment" status bypass
+--------------------------
 
-If everything was right (correct partner name, right amount) odoo will
-do the reconciliations **automatically**.
+It is possible to bypass the "In Payment" status of an invoice or bill if needed. Go to
+:menuselection:`Accounting --> Configuration --> Journals`, click on the **Cash** journal, and enter
+the :guilabel:`Outgoing Payments` tab. Then, click on the optional column button, check
```

Comment:
```suggestion
the :guilabel:`Outgoing Payments` tab. Then, click on the optional column button (:guilabel:`⋮`), check
```
The 'Outgoing Payments` tab only exists from v15, yet you are targeting v14. On v14, you go to the Payments Configuration tab then there is an Outstanding Payments Account field.
Add the vertical ellipsis symbol.